simple_bloom_reachability_map.go

```diff
@@ -0,0 +1,43 @@
+package node
+
+import (
+	"bloom"
+	"fmt"
+)
+
+type SimpleBloomReachabilityMap struct {
+	f *bloom.BloomFilter
+}
+
+func NewSimpleBloomReachabilityMap() ReachabilityMap {
+	m := SimpleBloomReachabilityMap{f: bloom.New(1000, 4)}
+	return m
+}
+
+func (m SimpleBloomReachabilityMap) IsReachable(n NodeAddress) bool {
+	entry := []byte(n)
+	res := m.f.Test(entry)
+	return res
+}
+
+func (m SimpleBloomReachabilityMap) AddEntry(n NodeAddress) {
+	entry := []byte(n)
+	m.f.Add(entry)
```

Comment:
Is there a reason to do it inline? personally I think it's easier to read through when it's like this, is inline more widely used ?